jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -18,14 +18,111 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 'use strict';
 
-YUI.add('entity-details', function() {
+YUI.add('entity-details', function(Y) {
+
+  var utils = Y.namespace('juju.views.utils');
 
   juju.components.EntityDetails = React.createClass({
+    fetchSuccess: function(models) {
```

Comment:
I'll add them, but docblocks are not at all consistent in the React code right now. They aren't in `search-results.js`, which is on me, but it was never noted in any of the reviews on that code. They're in `charmbrowser.js`, but only for the functions that aren't lifecycle-related (which seems very reasonable to me).

We should settle on some guidelines for moving foward and we may want to think about a different format post-YUI, e.g., JSDoc.